src/FunctionApps/PITest_FunctionApp/IngestFunction/settings.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+
+class Settings:
+    hostname = os.environ.get("VDHSFTPHostname")
+    username = os.environ.get("VDHSFTPUsername")
+    password = os.environ.get("VDHSFTPPassword")
```

Comment:
nit: can we use the same NAMING_CONVENTION for all env vars?

Reply:
^^^ 💯  for this please